src/dev/mod.ts

```diff
@@ -83,28 +84,23 @@ export async function generate(directory: string, manifest: Manifest) {
 // This file SHOULD be checked into source version control.
 // This file is automatically updated during development when running \`dev.ts\`.
 
-${
-    routes.map((file, i) => `import * as $${i} from "./routes${file}";`).join(
-      "\n",
-    )
-  }
-${
-    islands.map((file, i) => `import * as $$${i} from "./islands${file}";`)
-      .join("\n")
-  }
+${routes
+  .map((file, i) => `import * as $${i} from "./routes${file}";`)
+  .join("\n")}
+${islands
+  .map((file, i) => `import * as $$${i} from "./islands${file}";`)
+  .join("\n")}
```

Comment:
This repo use the Deno formatter by default for code format, I did some tests and I don't think those lines needed to be formatted